spec/models/metric_spec.rb

```diff
@@ -427,7 +427,7 @@
     context "with a small environment and time_profile" do
       before(:each) do
         @vm1 = FactoryGirl.create(:vm_vmware)
-        @vm2 = FactoryGirl.create(:vm_vmware, :hardware => FactoryGirl.create(:hardware, :memory_mb => 4096, :cpu_sockets => 2))
+        @vm2 = FactoryGirl.create(:vm_vmware, :hardware => FactoryGirl.create(:hardware, :memory_mb => 4096, :cpu_sockets => 2, :cpu_total_cores => 2))
```

Comment:
You can use `FactoryGirl.create(:vm_vmware, :hardware => FactoryGirl.create(:hardware, :cpu2x2, :memory_mb => 4096))` to simplify the cores and sockets setup.